### PR TITLE
test and implementation of xml feature

### PR DESCRIPTION
**Pull request by skp96**
[Link to story](https://trello.com/c/jAFvJPtN/82-xml-response-feature-3)

Highlights:

- Created a `GetXmlResponseAction` class to handle `GET` requests to `xml_response` endpoint, and tested it
- Added tests to `HttpResponseBuilderTest`, `RequestParserTest`, and `RouterTest` to ensure the functionality of the feature is as expected

### CODE REVIEW

**Review by k3ntako — APPROVED** [review]

File: src/test/kotlin/HttpResponseBuilderTest.kt [hunk 1]
```diff
@@ -3,6 +3,7 @@ import kotlin.test.*
 import response.HttpResponseBuilder
 import httpstatus.HttpStatus
 import org.json.JSONObject
+import kotlin.math.exp
```
Comment (cdemyanovich) [inline src/test/kotlin/HttpResponseBuilderTest.kt:6]: Why is this import added?

Comment (skp96) [reply src/test/kotlin/HttpResponseBuilderTest.kt:6]: I believe it was auto imported. I have to figure out how to turn that feature off in Intellij
